src/components/Home/CustomDay/index.js

```diff
@@ -14,7 +14,7 @@ const CustomDay = (props) => {
       sameDate,
     } } = props;
 
-
+    
```

Comment:
looks like there is unneeded tabbing in this empty line

Reply:
i have removed it. thanks